tests/system/Language/LanguageTest.php

```diff
@@ -46,4 +65,24 @@ public function testGetLineFormatsMessage()
 
 	//--------------------------------------------------------------------
 
+	public function testGetLineArrayFormatsMessages()
+	{
+		// No intl extension? then we can't test this - go away....
+		if (! class_exists('\MessageFormatter')) {
```

Comment:
Please note that our styleguide asks for the opening braces be on their own lines.